docs/Installation Guide.md

```diff
@@ -236,6 +236,13 @@ In addition to this, you would also have to populate the database for django-cit
 
     python manage.py migrate
     python manage.py cities_light
+    
+if ImportError: No module named cities_light ,this means cities_light has not installed properly.
+For this perform the following steps:
+
+    pip uninstall django-cities-light
+    pip install django-cities-light==3.2.0
+    pip install -r requirements.txt
```

Comment:
@Himanshi-Khandelwal I don't understand this change. What is effectively happening is you are asking them to install from requirements.txt, then uninstall it, then install version 3.2.0 and then again install requirements.txt